systemd/jenkins.service

```diff
@@ -0,0 +1,142 @@
+#
+# This file is managed by systemd(1). Do NOT edit this file manually!
+# To override these settings, run:
+#
+#     systemctl edit @@ARTIFACTNAME@@
+#
+# For more information about drop-in files, see:
+#
+#     https://www.freedesktop.org/software/systemd/man/systemd.unit.html
+#
+
+[Unit]
+Description=Jenkins Continuous Integration Server
+Requires=network.target
+After=network.target
+
+[Service]
+Type=notify
+NotifyAccess=main
+ExecStart=/usr/bin/@@ARTIFACTNAME@@
+Restart=on-failure
+SuccessExitStatus=143
+
+# Configures the time to wait for start-up. If Jenkins does not signal start-up
+# completion within the configured time, the service will be considered failed
+# and will be shut down again. Takes a unit-less value in seconds, or a time span
+# value such as "5min 20s". Pass "infinity" to disable the timeout logic.
+#TimeoutStartSec=90
+
+# Unix account that runs the Jenkins daemon
+# Be careful when you change this, as you need to update the permissions of
+# $JENKINS_HOME, $JENKINS_LOG, and (if you have already run Jenkins)
+# $JENKINS_WEBROOT.
+User=@@ARTIFACTNAME@@
+Group=@@ARTIFACTNAME@@
+
+# Directory where Jenkins stores its configuration and workspaces
+Environment="JENKINS_HOME=/var/lib/@@ARTIFACTNAME@@"
```

Comment:
`Environment="JENKINS_HOME=%h"`
since user created have it at the same location according to pre-install scripts

Reply:
> since user created have it at the same location

But this is not guaranteed.

Reply:
Default package behavior is well known that means you may rely on it and in case someone wants to override it it will go with drop-ins or other customization